core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -923,11 +923,20 @@ public String toString() {
   public static final PropertyKey MASTER_HEARTBEAT_INTERVAL_MS =
       new Builder(Name.MASTER_HEARTBEAT_INTERVAL_MS)
           .setAlias(new String[]{"alluxio.master.heartbeat.interval.ms"})
-          .setDefaultValue("1sec")
+          .setDefaultValue("10sec")
           .setDescription("The interval between Alluxio masters' heartbeats.")
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.MASTER)
           .build();
+  public static final PropertyKey MASTER_HEARTBEAT_TIMEOUT_MS =
+      new Builder(Name.MASTER_HEARTBEAT_TIMEOUT_MS)
+          .setAlias(new String[]{"alluxio.master.heartbeat.timeout.ms"})
```

Comment:
`setAlias` is for backward compatibility. For newly added properties, we don't need this.